.travis.yml

```diff
@@ -1,6 +1,21 @@
 services:
   - docker
 
-script:
-  - docker build --target test --tag georgefrancis:test0.1 .
-  - docker run --mount type=bind,source="$(pwd)",target=/app --env TRELLO_KEY --env TRELLO_TKN --env TRELLO_BOARD georgefrancis:test0.1 .
\ No newline at end of file
+jobs:
+  include:
+    - stage: test
+      script:
+        - docker build --target test --tag devops-starter:test0.1 .
+        - docker run --mount type=bind,source="$(pwd)",target=/app --env TRELLO_KEY --env TRELLO_TKN --env TRELLO_BOARD devops-starter:test0.1 .
+    - stage: build-tag-push
+      if: branch = master AND type = push
+      script:
+        - echo "$DOCKER_PASS" | docker login --username georgefrancis --password-stdin
```

Comment:
It would be nice to pull out some of these values as environment variables, such as your dockerhub username, and the heroku app name